gammapy/spectrum/sherpa_utils.py

```diff
@@ -88,7 +88,7 @@ def __init__(self, fit):
     def _calc(self, data, model, *args, **kwargs):
         self.fit.calc_statval()
         # Sum likelihood over all observations
-        total_stat = np.sum(self.fit.statval, dtype=np.float64)
+        total_stat = np.sum([np.sum(v) for v in self.fit.statval], dtype=np.float64)
```

Comment:
I see, good point (different binning will definitely happen when doing joint fit with different instruments). I think it would be good to add this scenario to the regression tests. So we would need to add something [here](https://github.com/gammapy/gammapy/blob/master/gammapy/spectrum/tests/test_fit.py#L164). Since I know these tests quite well, it would be quick for me to do it. If you want to get into the gammapy test, however, you're very welcome to do so. I could give you some hints how to implement this test, if you wish.